### PR TITLE
Adds Klingoffs and Space Elves to the game as roundstart species

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Iamgoofball and Anonymous Spriter
add: Added two new alien races to the game, Klingoffs and Space Elves!
/:cl:

## **Klingoffs**

A species of warriors, locked in an endless civil war on their home planet over putting butter on toast butter side up or butter side down. Nanotrasen often hires them as security forces due to their hostile nature and lack of reluctance to put boots to asses during riots.

## **Space Elves**

Space Elves have been floating around the galaxy for ages, spreading their peace-and-love-unless-you-hurt-trees message, but Nanotrasen only just finally convinced the Space Elves that towercaps are not trees and therefore it is safe to chop them down. Space Elves are universally hated by everyone for being Space Elves. Fuck Space Elves.

These two races are:
1. Unique sprites
2. Not just "X animal but anthro"
3. Actually have some backstory
4. Will create fun and interesting roleplay situations

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 // BEGIN_INCLUDE
 #include "_maps\__MAP_DEFINES.dm"
-#include "_maps\tgstation2.dm"
+#include "_maps\runtimestation.dm"
```
Comment (ExcessiveUseOfCobblestone) [inline tgstation.dme]: is this on purpose?

**Comment by RandomMarine** [comment]
It might be hard for silicons to tell these apart from humans.

**Comment by ChemicalRascal** [comment]
@RandomMarine: _Good._ That might prompt interesting situations.

**Comment by MrPerson** [comment]
Claiming they have a backstory is being kinda disingenuous since they really don't. I mean neither do the lizards or even humans so it's not a showstopper, but come on, be real.

**Comment by Iamgoofball** [comment]
@MrPerson 
It's pretty clear in the description, Nanotrasen grabs the klingoffs from their planet where they're in an endless civil war over butter on toast sides to work as security, and the Space Elves only just now started working for NT because they finally agreed towercaps aren't trees.

**Comment by DorfGuy1** [comment]
Spess dorfs when?

**Comment by calzilla1** [comment]
Lets put some /tg/ in /tg/station

**Comment by Iamgoofball** [comment]
by killing all space elves

**Comment by AdamElTablawy** [comment]
it's butter side up btw

**Comment by Iamgoofball** [comment]
this is true

**Comment by MrPerson** [comment]
Just because you wrote a joke doesn't mean we're expected to take it seriously.

**Comment by ExcessiveUseOfCobblestone** [comment]
lore is not a joke

**Comment by Incoming5643** [comment]
It ain't april yet

**Comment by Iamgoofball** [comment]
@MrPerson This isn't a joke.

@Incoming5643 This isn't a joke.

**Comment by Kromgar** [comment]
But muh fetishes

**Comment by Niknakflak** [comment]
This is ironically bullshit

**Comment by MrPerson** [comment]
I didn't say the PR was a joke. I get it, you really want these because they're cool and/or the playerbase will like them because they're cool, that's fine. Merge it.

Just don't pretend your half-assed backstory is anything but a joke for people to read and laugh at.

**Comment by Mervill** [comment]
Was the TNG uniform already in the game? Unique to Klingoffs? The world needs to know

**Comment by Iamgoofball** [comment]
Already in the game

**Comment by FantasticFwoosh** [comment]
This might yet be more relevant to #21807 mutant race feature blacklisting, throw the clingon head & elf ears in with the human parts in there with the cat-beasts & angels. Ew.

> Yes it doesn't need to be a animal but we already have human sprites to do the job of a non distinctive (like angel wings stick out through clothes) 'anthro' race for minor mutant features like this. 

Nice joke but could use more content, the pointy ears i like (there are already legolas snooty ripoffs RPing from time to time) but the clingon head is too pale to see and vaguely looks like a part of _the female anatomy_. 

On all other accounts polish them up and port them to be mutant parts for later application, maybe ghost roles/chemical induced mutations/random mutations or something.

> @ChemicalRascal its exactly the same issue with 'catpeople', a hat & a suit and the AI is forced into having to check race by logs and even then the mutant race might not be defined correctly causing issues.

**Comment by Cheridan** [comment]
do elves have full customization?
brown elves are best elves

**Comment by Dorsisdwarf** [comment]
Actually yeah this would be good to put in mutant races so it finally has a use other than fukken furries / anime trash

**Comment by HankMoodyExperience** [comment]
C A N T I N A S T A T I O N 1 3

**Comment by Kromgar** [comment]
I support this because it isn't furry/scaly bullshit

**Comment by ExcessiveUseOfCobblestone** [comment]
my rating skills are impeccable

**Comment by Iamgoofball** [comment]
They're staying as species, and yes cheridan

**Comment by Niknakflak** [comment]
administrative asimov nightmare

**Comment by XDTM** [comment]
I'd be happier if they were at least semi-original races and not ripoffs tbh.

**Comment by Niknakflak** [comment]
thats the joke

**Comment by Kromgar** [comment]
Well i mean people with cat ears and anthros are so original right?

At least plasma men are original

**Comment by TechnoAlchemisto** [comment]
Hey, this is bad

**Comment by Iamgoofball** [comment]
@XDTM I'm not adding "X animal but anthro" races.

@TechnoAlchemisto Explain why

**Comment by TechnoAlchemisto** [comment]
Because I don't even think you think this is a good idea

**Comment by Nienhaus** [comment]
I like it.

**Comment by Iamgoofball** [comment]
I do think it's a good idea @TechnoAlchemisto as I've stated in the past I'm all for alien species that aren't just "X animal but anthro". These ones are actually related to tgstation's theme and tg culture memes

**Comment by ExcessiveUseOfCobblestone** [comment]
demand all botanists be space elves.

**Comment by PKPenguin321** [comment]
Plasma men = cool unique race, +++

Lizards = literally a humanoid animal that's controversial, but at least it's somewhat different from a human and has enough aesthetic choices to justify it, +

Cat men = literally just a human with cat ears and that's it. Causes pointless controversy, ---

These = like lizards but without the upsides I listed and so close to humans that they're practically the same, --

Do we really need these?

**Comment by ExcessiveUseOfCobblestone** [comment]
yes space elves are important

**Comment by optimumtact** [comment]
There is absolutely nothing about these reaces worth merging other than them having a unique sprite.

Do not pass go, do not collect 200 dollars, go straight to jail

**Comment by ChangelingRain** [comment]
RUBBER FOREHEAD ALIENS DO NOT GET MERGED

**Comment by ChangelingRain** [comment]
THEY'RE LITERALLY HUMANS WITH A "UNIQUE" HEAD SPRITE

**Comment by optimumtact** [comment]
We had a chat in coderbus and I decided we will try this out

**Comment by KorPhaeron** [comment]
Why are these their own species instead of using the mutant parts list?

**Comment by Iamgoofball** [comment]
There's no good way to disable individual parts right now and I don't want to let cat ears/tails get enabled if these are enabled.

**Comment by PKPenguin321** [comment]
Also the klingon's head definitely looks like the head of a penis

**Comment by Iamgoofball** [comment]
@PKPenguin321 to be fair it does in trek too

**Comment by PKPenguin321** [comment]
Sure but the trek ones have the forehead line thing visibly protruding outwards, the sprites on these are so small it lacks depth which makes it look like a urethra

**Comment by Iamgoofball** [comment]
I could ask my spriter to redo some of it for more visible features

**Comment by Grazyn** [comment]
How about make elves able to choose their skin color like lizards, this will make them stand out a lot. It also falls into the "color-coded elves" trope and will offer more RP

**Comment by KorPhaeron** [comment]
Goof thats a really dumb reason for not using the proper system, I'd put DNM on it if it didn't have it already

**Comment by Grazyn** [comment]
> Goof thats a really dumb reason for not using the proper system, I'd put DNM on it if it didn't have it already

It'a almost like this is a joke pull to mock neko lovers who just lost their snowflake race

**Comment by Iamgoofball** [comment]
No rainbow elves

I want it to be it's own species fuck that shitty snowflake mutant parts code

**Comment by ExcessiveUseOfCobblestone** [comment]
elves are not mutated humans.

how could you even possibly suggest such a thing.

**Comment by Iamgoofball** [comment]
That too yeah they're fucking elves not humans

**Comment by Iamgoofball** [comment]
Maybe we can add smaller elf ears to it for half-elf status and keep space elves

**Comment by ExcessiveUseOfCobblestone** [comment]
Why aren't half-elves just called Helf?

**Comment by XDTM** [comment]
Could we stop with both the animal races and the meme races and make cool races that aren't human reskins instead

**Comment by Iamgoofball** [comment]
@XDTM can't make the clothes fit then

this is the first "human reskin" added

**Comment by Iamgoofball** [comment]
NEW SPRITES

**Comment by TechnoAlchemisto** [comment]
K N O B  H E A D

**Comment by PKPenguin321** [comment]
you circumcised the klingon 
also the elf ears are like a pixel off from cat ears which we already have
👎

**Comment by Iamgoofball** [comment]
there's most definitely a difference, look harder you nerdlord
let me get you a bald screenshot

**Comment by PKPenguin321** [comment]
alright but considering 99% of these guys wont be bald they're basically just cat people

**Comment by ChangelingRain** [comment]
If you want to make a new race and not just use mutant bodyparts, *put some actual effort into it.*

**Comment by calzilla1** [comment]
>Implying Lizards aren't just that.

**Comment by Iamgoofball** [comment]
@Cheridan please re-open this

**Comment by Iamgoofball** [comment]
I did make an actual race, two of them in fact,  stop disregarding the work of the spriters just because it isn't furry crap.

**Comment by FantasticFwoosh** [comment]
Pity. I wanted to play some elf bowling using assistants as skittles

**Comment by Firecage** [comment]
I am glad this is closed. Goof, can't you instead try to add in something more original instead of openly ripping off things from Star Trek? Seriously, I thought we weren't doing this anymore?

That Star Trek themed Holodeck option is bad enough.

**Comment by PKPenguin321** [comment]
Stealing from star trek is bad but cat people that are lifted from half of all anime are fine

Hmmmmmmm

**Comment by Iamgoofball** [comment]
@firecage Star Trek at least fits the fucking theme of the game. And elves are a classic tg joke.

**Pull request closed without merging**